### PR TITLE
Restructure installation docs

### LINKED ISSUE #375: Restructure Appsody install documentation
Part of #257

### PR DESCRIPTION
**Pull request by helenmasters**
## Checklist

- [x] commit message follows [commit guidelines](https://github.com/appsody/website/blob/master/CONTRIBUTING.md#commit-message-guidelines).
- [x] website builds and runs in production - for information on how to test locally go [here](https://github.com/appsody/website/blob/master/DEVELOPMENT.md).

## Description
Complete restructure of the installation related documentation:
- Split installation docs into one page per OS
- Create installation overview page
- Make Getting Started a top level menu
- Add new pages to sidebar
- Edit AAD Windows instructions
- Move SELinux section to FAQ
- Add redirects for previous install page and getting started page

## Related Issues
Fixes #375 #401

### CODE REVIEW

File: content/docs/sidebar.yaml [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
+- title: Installing
   items:
-    - title: Installation
-      path: /docs/getting-started/installation
-    - title: Quick Start
-      path: /docs/getting-started/quick-start
+    - title: Overview
+      path: /docs/installing/installing-appsody
+    - title: macOS
+      path: /docs//installing/macos
```
Comment (SueChaplain) [inline content/docs/sidebar.yaml]: Is the extra "/" going to cause a problem here?

Comment (helenmasters) [reply content/docs/sidebar.yaml]: I'm not sure but I've pushed a commit that removes it.

**Comment by helenmasters** [comment]
@SueChaplain and @neeraj-laad - following an informal review with Sue & Esther this morning I've pushed up another set of changes to the Overview page.

The only thing stopping this PR being merged (if the rewording can be agreed of course!!!) is that the Testing doc link in the Overview points to a nonexistent doc at the current time.

It would be good to get this merged this week if possible, the Testing link can always be edited out and added in under a separate PR.

File: content/docs/installing/installing-appsody.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+## Platforms
+
+Appsody supports the following operating systems on x86 hardware:
+
+ - macOS&reg;
+ - Ubuntu
+ - Red Hat Enterprise Linux (RHEL)
+ - Windows&reg; - Windows 10 Professional and Windows 10 Enterprise editions only
+
+ To find out more details about the operating system versions that Appsody tests against, refer to [Testing]().
```
Comment (SueChaplain) [inline content/docs/installing/installing-appsody.md]: Did you decide where to put the testing information yet?

File: content/docs/installing/installing-appsody.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
+---
+title: Overview - Installing Appsody
+---
+
+# Installing Appsody
+
+The easiest way to install Appsody is to use the installers which are available for all supported platforms, except Windows&reg;.
```
Comment (SueChaplain) [inline content/docs/installing/installing-appsody.md]: ... installers, which ....

File: content/docs/installing/windows.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
+---
+title: Installing on Windows
+---
+
+# Installing on Windows
+
+> Appsody runs only on Windows 10 Professional and Windows 10 Enterprise editions. If you are running Windows 10 Enterprise with authentication through Azure Active Directory, follow the additional instructions in [Appsody and Docker Desktop on Windows 10](/docs/docker-windows-aad).
```
Comment (SueChaplain) [inline content/docs/installing/windows.md]: Still not sure about the AAD stuff being in a hidden file. I can remember us discussing a troubleshooting topic at one point, which might be a good place to put this. Maybe we already talked about that?

File: content/docs/installing/installing-appsody.md (hunk 2 above)
Comment (helenmasters) [reply content/docs/installing/installing-appsody.md]: No, I've only got as far as raising an issue https://github.com/appsody/appsody/issues/906. To get this PR merged as is (as this level of detail wasn't in the previous docs either) I'm going to remove this sentence and add it back in when Luke and/or I get the testing doc written.

File: content/docs/installing/installing-appsody.md (hunk 3 above)
Comment (helenmasters) [reply content/docs/installing/installing-appsody.md]: Fixed in next commit.

File: content/docs/installing/installing-appsody.md (hunk 2 above)
Comment (helenmasters) [reply content/docs/installing/installing-appsody.md]: Removed in next commit.

File: content/docs/installing/windows.md (hunk 4 above)
Comment (helenmasters) [reply content/docs/installing/windows.md]: We've touched on this but not settling on a definitive solution as such. I'd like to get this merged as is and then address moving the AAD stuff (as I don't want to restructure FAQ/Troubleshooting under this PR as that feels like a separate topic).

**Comment by SueChaplain** [comment]
@helenmasters - I understand why the pre-req section is changed .... am ok with it. @neeraj-laad are you going to approve so we can merge?

**Pull request merged**